gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/DisableSystemAnimations.groovy

```diff
@@ -0,0 +1,30 @@
+package com.novoda.gradle.command
+
+import org.gradle.api.tasks.TaskAction
+
+class DisableSystemAnimations extends AdbTask {
```

Comment:
This task and the one below can be actually the same task (they just flip a flag that can be received in the constructor). When I mentioned 2 tasks I meant in the extension (see above). We would like to avoid to change one of this task without changing its companion.